go/cmd/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	internal "github.com/Canhassi12/scylla-cloud-getting-started/Internal/commands"
+	"github.com/Canhassi12/scylla-cloud-getting-started/Internal/database"
+	"github.com/joho/godotenv"
+	"github.com/scylladb/gocqlx/v2"
+)
+
+func main() {
+	godotenv.Load(".env")
+	
+	session, err := database.New().Connect(); if err != nil {
+		println("error database connection =>", err.Error())
+		return
+	}
+
+	err = executeMigrateFile(session, "./Internal/database/migrations/migrate.cql"); if err != nil {
+		println(err.Error())
```

Comment:
Here, error handling...

```go
fmt.Fprintf(os.Stderr, "error: %v\n", err)
os.Exit(1)
```

and remove the return at line 25. Check for all other errors that STOP your application and to the same change.